IDE visual updates (#8265)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/studio-ide/ide-user-interface.md

Reviewer: ```suggestion
3. [**Version Control**](#editing-features) &mdash; The <Constant name="cloud_ide" />'s powerful Version control section contains all git-related elements, including the <Constant name="git" /> actions button and the **Changes** section. 
```

Reviewer: i dont think we need a constant here - it should just be file explorer like the ui, not file catalog (which is what appears).
```suggestion
4. **File explorer &mdash;** The File explorer shows the filetree of your repository. You can:
```

3. **Build button &mdash;** The build button allows users to quickly access dbt commands related to the active model in the file editor. The available commands include dbt build, dbt test, and dbt run, with options to include only the current resource, the resource and its upstream dependencies, the resource, and its downstream dependencies, or the resource with all dependencies. This menu is available for all executable nodes.

4. **Lint button** &mdash; The **Lint** button runs the [linter](/docs/cloud/studio-ide/lint-format) on the active file in the file editor. The linter checks for syntax errors and style issues in your code and displays the results in the **Code quality** tab.

> 5. **dbt Copilot** &mdash; [dbt Copilot](/docs/cloud/dbt-copilot) is a powerful artificial intelligence engine that can generate documentation, tests, and semantic models for you. <Lifecycle status="self_service,managed,managed_plus" />

Reviewer: ```suggestion
5. **dbt Copilot** &mdash; [dbt Copilot](/docs/cloud/dbt-copilot) is a powerful artificial intelligence engine that generates documentation, data-tests, metrics, and semantic models for you. <Lifecycle status="self_service,managed,managed_plus" />
```

6. **Commands tab** &mdash; View the most recently run [dbt commands](/reference/dbt-commands) from your current IDE session, their results, and relevant system logs.

> 7. **Problems tab** &mdash; You must be running the <Constant name="fusion_engine" /> to utilize the problems tab. Gain insights into problems with your dbt project that may preven it from running properly in <Constant name="fusion" /> as you edit and before you execute runs. 

Author: ```suggestion
7. **Problems tab** &mdash; You must be running the <Constant name="fusion_engine" /> to utilize the problems tab. Gain insights into problems with your dbt project that may prevent it from running properly in <Constant name="fusion" /> as you edit and before you execute runs. 
```

<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-problems-tab.png" width="90%" title="Preview results show up in the Results console tab"/>

8. **Results tab** &mdash; The Results console tab displays the most recent Preview results in tabular format. 
<Lightbox src="/img/docs/dbt-cloud/cloud-ide/results-console-tab.png" width="90%" title="Preview results show up in the Results console tab"/>

> 9. **Code quality tab** &mdash; The Code Quality tab displays the results of the linter on the active file in the File Editor. It allows you to view code errors, provides code quality visibility and management, and displays the SQLFluff version used.

Reviewer: ```suggestion
9. **Code quality tab** &mdash; The Code quality tab displays the results of the linter on the active file in the File editor. It allows you to view code errors, provides code quality visibility and management, and displays the SQLFluff version used.
```
